GVRf/Sample/gvr-vuforia/src/org/gearvrf/vuforiasample/VuforiaSampleScript.java

```diff
@@ -20,157 +13,156 @@
 import com.qualcomm.vuforia.Trackable;
 import com.qualcomm.vuforia.TrackableResult;
 
+import org.gearvrf.GVRAndroidResource;
+import org.gearvrf.GVRContext;
+import org.gearvrf.GVRMaterial.GVRShaderType;
+import org.gearvrf.GVRMaterial;
+import org.gearvrf.GVRMesh;
+import org.gearvrf.GVRRenderData;
+import org.gearvrf.GVRRenderTexture;
+import org.gearvrf.GVRScene;
+import org.gearvrf.GVRSceneObject;
+import org.gearvrf.GVRScript;
+import org.gearvrf.GVRTexture;
+
 public class VuforiaSampleScript extends GVRScript {
 
-    private static final float CUBE_WIDTH = 20.0f;
-    private static final float SCALE_FACTOR = 5.0f;
-    private static final float BACKGROUND_OPACITY = 0.1f;
+    private static final String TAG = "gvr-vuforia";
+
     private GVRContext mGVRContext = null;
-    private GVRSceneObject bunny = null;
+    private GVRSceneObject teapot = null;
```

Comment:
teapot => mTeapot, we are not strictly enforcing m* for the member variables, but it would be consistent either way in the same code file.